integration/cri-o/crio_skip_tests.sh

```diff
@@ -8,36 +8,206 @@
 # TODO: the list of files is hardcoded for now, as we don't have listed all tests
 # that need to be skipped, but ultimately we will want to use *.bats
 # The file name is provided without its ".bats" extension for easier processing.
-declare -a bats_files_list=("ctr")
-
+declare -a bats_files_list=(
+    "ctr"
+    "cgroups"
+    "command"
+    "config"
+    "config_migrate"
+    "crio_wipe"
+    "ctr_seccomp"
+    "default_mounts"
+    "devices"
+    "drop_infra"
+    "hooks"
+    "image"
+    "image_remove"
+    "image_volume"
+    "infra_ctr_cpuset"
+    "inspect"
+    "inspecti"
+    "metrics"
+    "namespaces"
+    "network"
+    "network_ping"
+    "pod"
+    "profile"
+    "reload_config"
+    "reload_system"
+    "restore"
+    "runtimeversion"
+    "sanity_checks"
+    "selinux"
+    "shm"
+    "shm_size"
+    "stats"
+    "status"
+    )
 # We keep two lists for each bats file.
 # - [name]_skipCRIOTests for tests that we need to skip because of failures
 # - [name]_fixedInCrioVersion for tests that we need to skip when running
 #   with a version of cri-o that doesn't have the fix
 #
 # In both case, [name] is the basename of the bats file containing the tests.
 
+#This issue($url) - tracking crio integration test cases are skipped.
+#url=https://github.com/kata-containers/tests/issues/4459
```

Comment:
This line should not be commented out, right?
The goal is to have the URL showing up on each skipped test in the CI logs.

Reply:
You are right, updated, Thanks ):